### DISCUSSION TITLE
Feature comparison between drone and woodpecker

### DISCUSSION THREAD

**Discussion by mqus** [woodpecker-ci/woodpecker#4344] [Ask a Question]
Hi! I'm currently using drone and am considering a switch to woodpecker on my gitea instance. Is there something like a feature comparison, like things one would lose and things one would gain? 
In the faq it only mentions governance and licensing, which is a part of it, but this on its own is not that important to me.

**Comment by zc-devs** [comment]
> considering a switch to woodpecker

Why? What's wrong with Drone?

> feature comparison

I'm not aware of. Comparison to [which edition](https://docs.drone.io/enterprise/#what-is-the-difference-between-open-source-and-enterprise)?

> faq it only mentions governance and licensing
> Drone is published under Polyform Small Business license

 Which is kinda [misleading](https://docs.drone.io/enterprise/#is-drone-open-source).

---

> things one would lose and things one would gain

Make a PR to the docs with a table and features you want to see/compare, fill the `Drone` column and we will help you with `Woodpecker` one. What do you think?

**Reply by mqus** [reply]
> Why? What's wrong with Drone?

Afaict the momentum seems to be more on the woodpecker side and there are a few edges in drone. But this is just a feeling. Currently I am not using drone all that much but now have a few more uses for a proper CI and want to decide on something that sticks.

> Comparison to [which edition](https://docs.drone.io/enterprise/#what-is-the-difference-between-open-source-and-enterprise)?

Probably the enterprise edition, since the open source one does not seem to work other than for the most basic usecases (e.g. no more than a single runner on the same machine). I'm also using this one.

> Make a PR to the docs with a table and features you want to see/compare [...] What do you think?

That's a great idea! Maybe I would rather want to focus on the differences and not have two columns filled with checkmarks. But let's see how it works out. I might borrow from the comparison page you linked to though.

Thanks!
